app/src/test/java/echo/server/EchoServerTest.java

```diff
@@ -0,0 +1,16 @@
+package echo.server;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class EchoServerTest {
+  @Test
+  void testServerSocket() throws IOException {
```

Comment:
so unlike in python, we don't _have_ to use the word "test" at the front for our junit test :) Some people still do, but i feel like that's an extra word we that can just be taken out. the @test annotation describes that it's a test already imo. it might just be a personal preference though. maybe the other mentors have other opinions around this.

Reply:
also, just FYI, i think it's common in junit to underscores in between certain groupings.  Here are some examples i found . it lists some "cons" for each naming convention and that might help you form an opinion on the style you prefer.

https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea

Reply:
omg no more using test int he name! yay!!!!